pom.xml

```diff
@@ -33,8 +33,8 @@
         <logback.version>1.1.8</logback.version>
         <slf4j.version>1.7.22</slf4j.version>
 
-        <powsybl.core.version>3.2.0</powsybl.core.version>
-        <powsybl.rte-core.version>2.9.0</powsybl.rte-core.version>
+        <powsybl.core.version>3.3.0-SNAPSHOT</powsybl.core.version>
```

Comment:
You cannot change the ````powsybl-core```` version and the ````powsybl-rte-core```` version for all the tutorials. If it could be done with maven, only change the versions for the sensitivity tutorial.